esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -97,6 +100,11 @@
             ROSE_TASK_APP = compare
             RECIPE_NAME = "${CYLC_TASK_PARAM_fast##*--}"
             KGO_ROOT_PATH = {{ KGO_ROOT_PATH }}
+            # Create a variable that defines the path to the compare script.
```

Comment:
Do you think we could we get away with not having this comment? Is the name of the variable below descriptive enough that the comment is not necessary? 🤔

Reply:
I did wonder about this and i agree. Removed in commit bba02dfec :slightly_smiling_face: